lua/luasnip/util/path.lua

```diff
@@ -106,12 +94,12 @@ function Path.scandir(root)
 			elseif type == "directory" then
 				table.insert(dirs, path)
 			elseif type == "link" then
-				local followed_path
-				followed_path, type = resolve_symlink(path)
+				local followed_path = uv.fs_realpath(path)
+				local stat = uv.fs_stat(path)
```

Comment:
This should be `followed_path` instead of `path`, right?

Reply:
My bad.